src/apps/vpn/ui/screens/subscriptionNeeded/ViewSubscriptionNeededIAP.qml

```diff
@@ -192,4 +189,8 @@ VPNFlickable {
             }
         }
     }
+
+    Component.onCompleted: {
+        VPN.scheduleRefreshDataTasks(false)
+    }
```

Comment:
Looking into finding a better place to execute `TaskGetLocation` because if it takes a long time, this could result in the subscription option strings changing. Should probably do it after sign in, before the IAP screen is showed (during the loader)

Reply:
remove that and add this task everywhere we spawn a TaskProducts in mozillavpn.cpp